### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by dxp10** [nurullah-arican/SolidPrinciples-DependencyInjection#1]
Hi friend, could you rate my repositories and I'll rate yours at the same time? Thanks!

**Comment by nurullah-arican** [comment]
Of course. If there is a place where you want to add any addition, please send it to me. In this way, we can improve our codes.

**Comment by dxp10** [comment]
Okay, I'll look through your repositories just a little bit later. If there are any ideas or additions to your project I'll be sure to let you know!
